cmd/rook/noobaa/noobaa.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package noobaa defines rook-noobaa commands, mainly - the operator.
+package noobaa
+
+//
+//
+
+import (
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/operator/noobaa"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "cmd/rook/noobaa")
+
+// Cmd is the parent command for noobaa operator and tools
+var Cmd = &cobra.Command{
+	Use:   "noobaa",
+	Short: "Main command for noobaa operator.",
+	Long:  "See https://github.com/noobaa/noobaa-core and https://github.com/rook/rook",
+}
+
+// ASCIILogo is noobaa's logo ascii art
+const ASCIILogo = `
```

Comment:
Normally the `rook` command is not run directly, so why print the logo?

Personally speaking, printing a logo in the logs just means unnecessary lines of logs (e.g., to be shipped to Elasticsearch).
Though I think I know where you are coming from with this.

Reply:
It is printed once on operator startup so I don't think it matters to anyone.
The reason to print a logo is ... well there is no reason, it is just beautiful.
I prefer to keep it for now, but will certainly remove it if others will complain.
Thanks! Resolving.